packages/drawer/src/Drawer.tsx

```diff
@@ -52,17 +53,19 @@ export type DrawerProps = PropsWithChildren<{
    * @defaultValue `false`
    */
   showCloseButton?: boolean
-}>
+}
+export type DrawerProps = PropsWithChildren<Props>
 
 export default function Drawer({
-  ['aria-label']: ariaLabel,
+  'aria-label': ariaLabel,
   children,
   closeButtonTitle = 'Close',
   disableBackdropClick = false,
   disableEscapeKeyDown = false,
   onClose,
   open = false,
   showCloseButton = false,
+  sx,
```

Comment:
Can we also receive `className`That would allow us to customize the component with the `styled` HOC.

```
cost CustomDrawer = styled(Drawer)`
  // New styles
`
```

If we do that, do we also need to support `sx`?

Reply:
Definitely would support utilizing styled-components over inline styles

Reply:
@huhgawz  Not sure that is possible since this component returns a `Portal`

Reply:
We'll apply the styles to `DrawerBase` just like you do with `sx`. Seems unlikely that we'll style the Backdrop as well. 

So what matters is where you pass the `className` to, not what is the outer wrapper. Optionally we could add an extra wrapper to target both the Backdrop and the DrawerBase, but that doesn't seem like a common use case for us.

Reply:
Fixed in 592e50604bb9c18cd66a7cd3cc270672ad6ed12f